check/build-changed-protos

```diff
@@ -55,24 +55,16 @@ else
 fi
 
 # All the protos.
+echo "Looking for protos in $(pwd)"
+
 protos=$(find -type f -name "*.proto" -printf '%P\n' \
     | grep "google/api/.*\.proto$" \
     | sed -e "s/\.proto$//")
 
-# Get the changed protos and strip them of ".proto"
-changed=$(git diff --name-only ${rev} -- \
-    | grep "google/api/.*\.proto$" \
-    | sed -e "s/\.proto$//"
-)
-if [[ ! -z "$changed" ]]; then
-    for base in $changed
-    do
-        echo -e "Found changed proto: '${base}.proto'"
-    done
-fi
+echo "Found protos: ${protos}"
 
 # We potentially will attempt to build a second time.
-for base in $changed
+for base in $protos
 do
     python -m grpc_tools.protoc -I=cirq-google --python_out=cirq-google --mypy_out=cirq-google ${base}.proto
```

Comment:
I'd prefer to invoke `dev_tools/build-protos.sh` directly here, rather than duplicating some of the logic. In particular, the build-protos.sh script has an additional step to update some imports in the files: https://github.com/quantumlib/Cirq/blob/master/dev_tools/build-protos.sh#L41, so I would expect that running this CI check should result in some diffs between the files it generates and what build-protos.sh has generated and is checked in.

It's possible that the extra import handling step is no longer needed with newer versions of the protoc compiler, so we might be able to remove that, but I'd still prefer to invoke the same script here as what we expect users to do before checking in proto changes, since the whole point of this check is to ensure that the protos were generated in the expected way.

Reply:
Done, I think. It does indeed revert the changes to the generated files. Thank you.